.gitignore

```diff
@@ -58,6 +58,7 @@ junit.xml
 .hypothesis/
 .pytest_cache/
 cover/
+logs/containerized-tests
```

Comment:
Ah, I thought logs/ was already ignored, but I see it's covered by `*.log`
In this case, it seems we're saving an .xml file, which is already ignored. 
So the additional ignore shouldn't be needed, but we can instead just ignore the entire logs/ folder for clarity